src/GLRenderObject.jl

```diff
@@ -117,8 +115,8 @@ prerendertype{Pre}(::Type{RenderObject{Pre}}) = Pre
 prerendertype{Pre}(::RenderObject{Pre}) = Pre
 
 extract_renderable(context::Vector{RenderObject}) = context
-extract_renderable(context::RenderObject) = RenderObject[context]
-extract_renderable{T <: Composable}(context::Vector{T}) = map(extract_renderable, context)
+extract_renderable(context::Re(erObject[context]
```

Comment:
there are more, but they look all like the parser get out of sync at some point and just produces garbage from there on